internal/util/fileop.go

```diff
@@ -166,22 +166,50 @@ func (this *FileOperation) Exec(baseDir string) error {
 	parts := []string(*this)
 	command := parts[0]
 
+	var errs error = nil
 	if command == FILE_OP_LONG_COPY {
-		sourcePath := resolvePath(parts[1], baseDir, false)
+		sourcePathGlob := resolvePath(parts[1], baseDir, false)
 		destPath := resolvePath(parts[2], baseDir, false)
-		if sourcePath == destPath {
-			return nil
+
+		sourcePaths, err := prepForGlobs(sourcePathGlob, destPath)
+		if err != nil {
+			return fmt.Errorf("Failed to prep globs '%s' for a copy: '%w'.", sourcePathGlob, err)
+		}
+
+		for _, sourcePath := range sourcePaths {
+			if sourcePath == destPath {
+				continue
+			}
+
+			err = CopyDirent(sourcePath, destPath, false)
+			if err != nil {
+				errs = errors.Join(errs, err)
+			}
 		}
 
-		return CopyDirent(sourcePath, destPath, false)
+		return errs
 	} else if command == FILE_OP_LONG_MOVE {
-		sourcePath := resolvePath(parts[1], baseDir, false)
+		sourcePathGlob := resolvePath(parts[1], baseDir, false)
 		destPath := resolvePath(parts[2], baseDir, false)
-		if sourcePath == destPath {
-			return nil
+
+		sourcePaths, err := prepForGlobs(sourcePathGlob, destPath)
+		if err != nil {
+			return fmt.Errorf("Failed to prep globs '%s' for a move: '%w'.", sourcePathGlob, err)
+		}
+
+		for _, sourcePath := range sourcePaths {
```

Comment:
(sourcePath == destPath) is not handled.
Make sure there is a test case for this as well.